css/css-ui-3/text-overflow-017.html

```diff
@@ -0,0 +1,37 @@
+﻿<!DOCTYPE html>
+<html class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: interacting with the ellipsis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#text-overflow">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem interact dom">
+<meta name="assert" content="Ellipsing must not affect dispatching of pointer events.">
```

Comment:
“must not affect layout nor dispatching of pointer events” is sufficiently vague that I don't know how to review this test. It looks like you're testing that clicking on an ellipsis has the same effect as clicking on the block itself. That's interesting, but what about if the elided text was in a SPAN? Should the event get dispatched to the block or to the SPAN?